src/components/learner-credit-management/tests/BudgetDetailPage.test.jsx

```diff
@@ -701,4 +701,69 @@ describe('<BudgetDetailPage />', () => {
 
     expect(screen.getByText('loading budget activity overview')).toBeInTheDocument();
   });
+
+  it('displays bulk actions with remind when row selected and state allocated', () => {
+    useSubsidyAccessPolicy.mockReturnValue({
+      isInitialLoading: false,
+      data: {
+        uuid: 'a52e6548-649f-4576-b73f-c5c2bee25e9c',
+        policyType: 'AssignedLearnerCreditAccessPolicy',
+        isAssignable: true,
+      },
```

Comment:
nit: could re-use `mockAssignableSubsidyAccessPolicy` here instead of defining your own assignable subsidy access policy.